cypress/e2e/backoffice/file-manager-attachment/file-list.cy.ts

```diff
@@ -0,0 +1,79 @@
+        const fileManagerAttachmentAddPage = container.get(FileManagerAttachmentAddPage);
+        
+        // Prepare test files that meet the constraints
+        const testFiles: { fileContent: any; fileName: string; mimeType: string; filePath: string; }[] = [];
+        
+        cy.fixture('suite/backoffice/file-manager-attachment/test-files/document1.pdf', 'binary').then((fileContent) => {
+            testFiles.push({ fileContent, fileName: 'document1.pdf', mimeType: 'application/pdf', filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/document1.pdf' });
+        }).then(() => {
+            cy.fixture('suite/backoffice/file-manager-attachment/test-files/image1.jpeg', 'binary');
+        }).then((fileContent) => {
+            testFiles.push({ fileContent, fileName: 'image1.jpeg', mimeType: 'image/jpeg', filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/image1.jpeg' });
+        }).then(() => {
+            cy.fixture('suite/backoffice/file-manager-attachment/test-files/image2.png', 'binary');
+        }).then((fileContent) => {
+            testFiles.push({ fileContent, fileName: 'image2.png', mimeType: 'image/png', filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/image2.png' });
+        }).then(() => {
+            cy.fixture('suite/backoffice/file-manager-attachment/test-files/document2.pdf', 'binary');
+        }).then((fileContent) => {
+            testFiles.push({ fileContent, fileName: 'document2.pdf', mimeType: 'application/pdf', filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/document2.pdf' });
+        }).then(() => {
+            fileManagerAttachmentAddPage.visit();
+            fileManagerAttachmentAddPage.uploadFiles(testFiles);
+            fileManagerAttachmentAddPage.verifyFileUploadConstraints();
+            fileManagerAttachmentAddPage.submitForm();
+            fileManagerAttachmentAddPage.verifySuccessMessage();
+        });
+    });
+
+    it.skip('should successfully delete a file', () => {
+        const fileManagerAttachmentDeletePage = container.get(FileManagerAttachmentDeletePage);
+
+        fileManagerAttachmentListPage.visit();
+        fileManagerAttachmentListPage.clickDeleteButton();
+        
+        fileManagerAttachmentDeletePage.confirmDelete();
+        fileManagerAttachmentDeletePage.verifySuccessMessage();
+    });
+
+    it.skip('should display file details on view page', () => {
+        const fileManagerAttachmentViewPage = container.get(FileManagerAttachmentViewPage);
+
+        fileManagerAttachmentListPage.visit();
+        fileManagerAttachmentListPage.clickViewButton();
+        fileManagerAttachmentViewPage.verifyFileDetailsAreVisible();
+    });
+});
+
+function describeWithoutMp(title: string, options: { tags: string[] }, fn: () => void): void {
+    (['b2c-mp', 'b2b-mp'].includes(Cypress.env('repositoryId')) ? describe : describe.skip)(title, fn);
```

Comment:
something is fishy here, we need non-split and b2b_mp in the future, but we don't need b2c_mp and yet we don't have b2b-mp